docs/_data/importer_dependencies.yml

```diff
@@ -0,0 +1,109 @@
+---
+behance:
+- behance
+- safe_yaml
+blogger:
+- rexml/document
+- rexml/parsers/streamparser
+- rexml/streamlistener
+- safe_yaml
+dotclear:
+- active_support
+- active_support/core_ext/string/inflections
```

Comment:
Should we normalize this to the gem it comes from? I wonder if there's a way to find the file using Ruby's $LOAD_PATH and normalize on the gem name. For example, this and the line above are both from the gem "activesupport", I believe.

Reply:
I thought about it but dropped it for a future pull request..

Reply:
Awesome suggestion regarding the use of `$LOAD_PATH`.. I did not think of that..

Reply:
Gotcha. This doesn't quite answer the question of how to install the necessary dependencies for the average end user who isn't that familiar with ruby. You and I know how to install activesupport properly but they may not. I wonder how much this change helps without a more explicit instruction like:


```
To use this importer, run the following command beforehand:

    gem install activesupport rexml sequel sqlite3
```

It currently reads to me like a maintainer documentation change rather than an end-user documentation change 🤔

Reply:
I see.. I didn't add the `gem install` so that users do not spend time installing gems they *may* already have installed..
But yes, your point is valid and makes sense.